apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -193,17 +193,15 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             type: Virtual,
             isRequired: false,
             graphQLReturnType: 'Boolean',
-            resolver: async (receipt, _, context) => {
-                const receipts = await BillingReceiptApi.getAll(context, {
+            resolver: async (receipt) => {
+                const receipts = await find('BillingReceipt', {
```

Comment:
Are you sure you won't get a lot of data here?

Reply:
first 1

Reply:
`find` has not prop `first`